### PR TITLE
Add types aliases used in flet into `aliases.md`.

### PR DESCRIPTION
**Pull request by taaaf11**
Add types aliases used in flet into `aliases.md`. For types that have information in `/docs/reference/types`, I have added a link to them. But I have left linking `Animation`, `Offset`, `Rotate` and `Scale` as they are not present there.

### CODE REVIEW

File: docs/reference/types/aliases.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Union of `int`, `float`, [`Margin`](/docs/reference/types/margin) and `None`
+
+### `Number`
+
+Union of `int`, `float`
+
+### `OffsetValue`
+
+Union of `Offset`, `None`, and union of two; two valued tuples of 1st element of type `float` and 2nd of type `int`
```
Comment (ndonkoHenri) [inline docs/reference/types/aliases.md:31]: Can you add a link to the offset page? (Just Like you did for border radius)

File: docs/reference/types/aliases.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Union of `dict` of `str` and [`Number`](#number), and [`Number`](#number).
+
+### `RotateValue`
+
+Union of `int`, `float`, `Rotate` and `None`
+
+### `ScaleValue`
+
+Union of `int`, `float`, `Scale` and `None`
```
Comment (ndonkoHenri) [inline docs/reference/types/aliases.md:59]: Here too, a link to Scale class.

**Review by ndonkoHenri — CHANGES_REQUESTED** [review]

**Comment by ndonkoHenri** [comment]
Thanks again.
I am just from noticing that we lack these types you mentioned (rotate, scale, ...).
You can add them if you wish, simply by following the already present types.

Also, please don't forget to mention this aliases where possible. For example, all the padding-related props should have "Value is of type `PaddingValue`...", same for the border-radius-related props, etc.

**Comment by taaaf11** [comment]
@ndonkoHenri 
I have "copied" the documentation from flutter docs (`Transform` widget docs).

Also, I have left the `alignment` property of `Rotate` class because I had no clue what would the property do.

File: docs/reference/types/scale.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+title: Scale
+sidebar_label: Scale
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+`Scale` has the following properties:
+
+### `scale_x`
```
Comment (ndonkoHenri) [inline docs/reference/types/scale.md:11]: Can you please order the props in alphabetical order?

**Review by ndonkoHenri — CHANGES_REQUESTED** [review]

**Comment by ndonkoHenri** [comment]
Thanks again for your efforts!

> Also, please don't forget to mention this aliases where possible. For example, all the padding-related props should have "Value is of type PaddingValue...", same for the border-radius-related props, etc.

If you have time, you could also do this. 🙂

**Comment by ndonkoHenri** [comment]
By the way, you need to include the new files in the [sidebar.js](https://github.com/flet-dev/website/blob/main/sidebars.js) for them to be visible on the website.

**Comment by taaaf11** [comment]
yes. I was thinking about how the files in the `types` folder go into their specific "types" in the documentation on web. Didn't know it was `sidebar.js` (by the way I don't know js)

**Comment by taaaf11** [comment]
Should I mention type aliases of many controls in this same pull request. I was thinking there could be pull requests one after the other.

**Comment by taaaf11** [comment]
Okay, so merge this pr. I will do remaining in next prs

**Review by ndonkoHenri — APPROVED** [review]

**Pull request merged**